core/src/main/scala/org/broadinstitute/workbench/hamm/dao/WorkflowMetadataDAO.scala

```diff
@@ -3,11 +3,12 @@ package org.broadinstitute.workbench.hamm.dao
 import cats.effect.Sync
 import org.broadinstitute.workbench.hamm.model.CromwellMetadataJsonCodec._
 import org.broadinstitute.workbench.hamm.model.{MetadataResponse, WorkflowId}
+import org.http4s.Uri
 import org.http4s.circe.CirceEntityDecoder._
 import org.http4s.client.Client
 
-class WorkflowMetadataDAO[F[_]: Sync](httpClient: Client[F]) {
+class WorkflowMetadataDAO[F[_]: Sync](httpClient: Client[F], uri: Uri) {
   def getMetadata(workflowId: WorkflowId): F[MetadataResponse] = {
-    httpClient.expect[MetadataResponse](s"https://cromwell.dsde-alpha.broadinstitute.org/api/workflows/v1/${workflowId.uuid}/metadata") //TODO: figure out how to get the url properly
+    httpClient.expect[MetadataResponse](uri + s"/${workflowId.uuid}/metadata") //TODO: figure out how to get the url properly
```

Comment:
I'd prefer to only move the host name to config and leave the `/api/workflows/v1/` part here just like in other places